langchain4j/src/main/java/dev/langchain4j/memory/chat/MessageWindowChatMemory.java

```diff
@@ -33,20 +34,22 @@
  * to avoid problems with some LLM providers (such as OpenAI)
  * that prohibit sending orphan {@code ToolExecutionResultMessage}(s) in the request.
  * <p>
- * The state of chat memory is stored in {@link ChatMemoryStore} ({@link InMemoryChatMemoryStore} is used by default).
+ * The state of chat memory is stored in {@link ChatMemoryStore} ({@link SingleSlotChatMemoryStore} is used by default).
  */
-public class MessageWindowChatMemory implements ChatMemory {
+public class MessageWindowChatMemory implements ChatMemory, RemovableChatMemory {
 
     private static final Logger log = LoggerFactory.getLogger(MessageWindowChatMemory.class);
 
     private final Object id;
     private final Integer maxMessages;
     private final ChatMemoryStore store;
 
+    private Consumer<Object> memoryCleaner;
```

Comment:
```suggestion
    private Consumer<Object> memoryRemover;
```

To keep consistency with `RemovableChatMemory`, WDYT?